src/main/java/frc/robot/commands/EndGameClimbMid.java

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+import frc.robot.RobotContainer;
+
+// NOTE:  Consider using this command inline, rather than writing a subclass.  For more
+// information, see:
+// https://docs.wpilib.org/en/stable/docs/software/commandbased/convenience-features.html
+public class EndGameClimbMid extends SequentialCommandGroup {
+  /** Creates a new EndGameClimbMid. */
+  public EndGameClimbMid() {
+    // Add your commands in the addCommands() call, e.g.
+    // addCommands(new FooCommand(), new BarCommand());
+    addCommands(
+      new ClimberGoTo(0),
+      new IntakeGoTo(-2),
+      new WaitCommand(0.5),
+      new ClimberGoTo(8)
+      //MID BAR JUST INTAKE ACTUATORS
+    );
+  }
+  @Override
+  public void end(boolean interrupted) {
+    RobotContainer.m_intakeSubsystem.stopActuators();
+    RobotContainer.m_climberSubsystem.stop();
```

Comment:
You should NOT be trying to control a subsystem by calling back thru RobotContainer!

The subsystems you need should be passed into the Command as inputs, saved in private members and then referenced via private members of the Command.